pii_data.py

```diff
@@ -31,7 +31,11 @@ def has_credit_card(self):
         return True if re.search(r'(\d{4}-\d{4}-\d{4}-\d{4})|(\d{4}-\d{6}-\d{5})', self) else None
 
     def has_at_handle(self):
-        return None
+        match = re.search('\@', self)
```

Comment:
The problem is this will match anything with an @ symbol - joe@gmail.com or @Joe - you need to make this more specific so it won't match an email.  You should go to https://regex101.com and try a regex such as this to see if it will work for you
```
(^|\s)@\w+
```